static/js/components/SelectWithChips.jsx

```diff
@@ -2,12 +2,12 @@ import React, { useState } from "react";
 import PropTypes from "prop-types";
 import makeStyles from "@mui/styles/makeStyles";
 import {
+  Chip,
   FormControl,
   InputLabel,
-  Select,
   MenuItem,
+  Select,
   useTheme,
```

Comment:
Same comment as before, please split in distinct imports for webpack splitting

Reply:
✅ Done.